core/tests/org.fusesource.ide.camel.tests.util/META-INF/MANIFEST.MF

```diff
@@ -6,6 +6,24 @@ Bundle-Version: 9.1.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Require-Bundle: org.junit;bundle-version="4.12.0",
  org.fusesource.ide.camel.model.service.core;bundle-version="9.1.0",
- org.fusesource.ide.camel.model.service.v2_17_3;bundle-version="9.1.0"
-Export-Package: org.fusesource.ide.camel.tests.util
+ org.fusesource.ide.camel.model.service.v2_17_3;bundle-version="9.1.0",
+ org.fusesource.ide.camel.model.service.core.tests.integration,
+ org.fusesource.ide.branding,
+ org.eclipse.core.runtime;bundle-version="3.11.1",
+ org.eclipse.core.resources,
+ org.eclipse.core.databinding;bundle-version="1.5.0",
+ org.eclipse.jdt.ui,
+ org.eclipse.swt,
```

Comment:
org.eclipse.swt is most of the time not a real required dependency and should be avoided